en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+  The operator can then test the new tuning while taking-off carefully.
+* `2`: apply immediately (default for fixed-fings).
+  The new tuning is applied, disturbances are sent to the controller and the stability is monitored during the next 4 seconds.
+  If the control loop is unstable, the control gains are immediately reverted back to their previous value.
+  If the test passes, the pilot can then use the new tuning.
+
+
+Fixed wing vehicles (only) can tune which axes are tuned using the [FW_AT_AXES](../advanced_config/parameter_reference.md#FW_AT_AXES) bitmask parameter:
+
+* bit `0`: roll (default)
+* bit `1`: pitch (default)
+* bit `2`: yaw
+
+
+### Troubleshooting
+
+#### The drone oscillates when performing the testing maneuvers prior to the multirotor auto-tuning
+
+* slow oscillations (1 oscillation per second or slower): this often occurs on large platforms and means that the attitude loop is too fast compared to the rate loop -> decrease `MC_[ROLL|PITCH]_P` by steps of 1.0.
+* fast oscillations (more than 1 oscillation per second): this is because the gain of the rate loop is too high -> decrease `MC_[ROLL|PITCH|YAW]RATE_K` by steps of 0.1
+
+#### The auto-tuning sequence fails
+
+If the drone was not moving enough during auto-tuning, the system identification algorithm might have issues to find the correct coefficients.
+Increase the `MC/FW_AT_SYS_ID` by steps of 1 and trigger the auto-tune again.
+
+#### The drone oscillates after auto-tuning
+
+Due to effects not included in the mathematical model such as delays, saturation, slew-rate, airframe flexibility, the loop gain can be too high.
+To fix this, reduce the `MC_[ROLL|PITCH|YAW]RATE_K` parameters by steps of 0.1 until the oscillation stops.
+If a slow oscillation appears, reduce `FW_[RPY]R_P` by steps of 0.5.
+
+
+### FAQ
+
+#### What frames types are supported?
+
+Autotuning is enabled for multicopter, fixed wing, and VTOL vehicles.
+
+While it is not yet enabled for other frame types, in theory it an be used with any frame that uses a rate controller.
+
+#### Does autotuning work for all supported airframes?
+
+The mathematical model used by autotuning to estimate the dynamics of the drone assumes this it is a linear system with no coupling between the axes (SISO), and with a limited complexity (2 poles and 2 zeros).
+If the real drone is too far from those conditions, the model will not be able to represent the real dynamics of the drone.
+
+In practise, autotuning generally works well for fixed wing and multicopter, provided the frame is not too flexible.
+
+#### How long does autotuning take?
+
+Tuning takes 5s-20s per axis (aborted if tuning could not be established in 20s) + 2s pause between each axis + 4s of testing if the new gains are applied in air.
+
+A multicopter must tune all three axes, and by default does not test the new gains in-air.
+The range is therefore between 19s (`5 + 2 + 5 + 2 + 5`) and 64s (`20x3 + 2x2`).
```

Comment:
@bresch You calculated the time for three axis as 3x time for 1 axis, and 3x space between axis. My thinking was that the space would actually be just between axes, and not afterwards (so 2x axis)? Or am I wrong here?

Reply:
There is a 2s delay between the last axis and the in-air test too, so it's really 3x time for 1 axis + 3x space between axis

Reply:
Yeah, but by default you don't run the in-air test on multicopter, and this case is "the default multicopter case". I will fix up the default fixed wing case though.